k8s/k8s.go

```diff
@@ -828,3 +867,30 @@ func getPodCidr(client *kubernetes.Clientset, conf types.NetConf, nodename strin
 	}
 	return node.Spec.PodCIDR, nil
 }
+
+// handleAzureIPAM mutates the given args to include the Azure IPAM subnet, if appropriate.
+func handleAzureIPAM(logger *logrus.Entry, args *skel.CmdArgs, filename string) error {
```

Comment:
`handle` always seems like a copy out :-P.  How about `adjustAzureIPAMConfig` or `fixUpAzureIPAMConfig` or something like that?